packages/scandipwa/src/component/Loader/Loader.component.js

```diff
@@ -22,11 +22,13 @@ import './Loader.style';
  */
 export class Loader extends PureComponent {
     static propTypes = {
-        isLoading: PropTypes.bool
+        isLoading: PropTypes.bool,
+        mix: PropTypes.shape({ block: PropTypes.string, elem: PropTypes.string })
```

Comment:
There is mixType  for PropTypes. Check Types folder

Reply:
true.. Done!